python/tests/test_json.py

```diff
@@ -9,6 +9,11 @@
 from pydantic import BaseModel
 
 
+def test_encode_json_recursively_encodes_tuples():
+    result = encode_json((np.float32(0.1), np.float32(0.2)), None)
```

Comment:
What's the `None` for? Incidentally making sure this function doesn't blow up when given empty values?

Reply:
It's a callback for uploading files which isn't used.

Reply:
Got it. https://github.com/replicate/cog/blob/2d86d20947d66fbb8870ada667f32f033508eea7/python/cog/json.py#L17:L18